utils/exporters/blender/addons/io_three/exporter/api/object.py

```diff
@@ -374,7 +378,7 @@ def matrix(obj, options):
         parent_inverted = obj.parent.matrix_world.inverted(mathutils.Matrix())
         return parent_inverted * obj.matrix_world
     else:
-        return AXIS_CONVERSION * obj.matrix_world
```

Comment:
@phfatmonkey  Second change in mesh.py:
+        vertices_.extend((vertex.co.x, vertex.co.z, -vertex.co.y))

is not in new exporter version, it still:
vertices_.extend((vertex.co.x, vertex.co.y, vertex.co.z))

and was generating 90º degrees rotate in mesh